rfc/0009-expressions/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+
+SPDX-License-Identifier: AGPL-3.0-only
+-->
+
+# RFC 0009: Simple Expressions
+
+| | |
+|---|---|
+| Feature Tag | `Simple Expressions` | 
+| Status | `DRAFT` | <!-- Possible values: DRAFT, DISCUSSION, ACCEPTED, REJECTED -->
+| Responsible | `georg-schwarz` | <!-- TODO: assign yourself as main driver of this RFC -->
+<!-- 
+  Status Overview:
+  - DRAFT: The RFC is not ready for a review and currently under change. Feel free to already ask for feedback on the structure and contents at this stage.
+  - DISCUSSION: The RFC is open for discussion. Usually, we open a PR to trigger discussions.
+  - ACCEPTED: The RFC was accepted. Create issues to prepare implementation of the RFC.
+  - REJECTED: The RFC was rejected. If another revision emerges, switch to status DRAFT.
+-->
+
+## Summary
+
+This RFC introduces a simple expression language for numeric and boolean values.
+
+## Motivation
+
+Right now, users have to to define property values of fractions in the decimal representation, e.g., `0.3`. They cannot express non--terminating decimal values or repeating decimal ones, e.g., `1/3` (a third). 
+
+For future features (like value transformations), we also need boolean expressions that allow for numeric comparisons, e.g. `3 > 5` should resolve to `false`.
+
+## Explanation
+
+### Operators
+
+The following operators are in descending order by their precedence:
+- `()` for grouping a sub-expression
+- unary operators (see below)
+- binary operators (see below)
+
+#### Unary Prefix Operatory
+Binary prefix operators follow the pattern `op a`. Brackets can be used for grouping larger sub-expressions.
+- `floor`, `ceil`, `round` for conversion of decimal numbers to integers.
+- `not` for unary inversion of boolean values
+
+#### Binary Operators
+Binary infix operators follow the infix pattern `a op b`.
+- `pow`, `root` for power and root calculation. E.g., `2 pow 3` is 2³, `5 root 2` is `sqrt(5).
```

Comment:
Imho we should directly include a `sqrt` unary operator for the very common square roots.